packages/ui/src/app/constants/currency.ts

```diff
@@ -0,0 +1,4 @@
+export const CurrencyName = {
+  integerValue: 'tJOY',
+  decimalsValue: 'tBDH',
```

Comment:
We should probably keep this an object since there might be some more denomination later on but in the currently chosen solution everything will be in `JOY/tJOY`, either `XXX.XX JOY` or `> 0.01 JOY`